File: src/BukkitPE/plugin/EventExecutor.php
<?php
namespace BukkitPE\plugin;

use BukkitPE\event\Event;
use BukkitPE\event\Listener;

interface EventExecutor{

	/**
	 * @param Listener $listener
	 * @param Event    $event
	 *
	 * @return void
	 */
	public function execute(Listener $listener, Event $event);
}

File: src/net/BukkitPE/plugin/EventExecutor.java
package net.BukkitPE.plugin;

import net.BukkitPE.event.Event;
import net.BukkitPE.event.Listener;

/**
 * author: iNevet
 * BukkitPE Project
 */
public interface EventExecutor {

    void execute(Listener listener, Event event);
}
